offline/docs.md

```diff
@@ -324,26 +324,30 @@ For full docs with details and explanations please see https://github.com/wireap
 First, make sure you have a certificate for `sftd.<yourdomain>`. This could be the same wildcard or SAN certificate
 you used at previous steps.
 
-If you want to restrict SFT to certain nodes, make sure that in your inventory
-you have annotated all the nodes that are able to run sftd workloads correctly.
+Next, copy the values/sftd/prod-example-values.yaml to values/sftd/values.yaml, and change the contents accordingly. 
+
+ * If your turn servers can be reached on their public IP by the SFT service, Wire recommends you enable cooperation between TURN and SFT. add a line reading `turnDiscoveryEnabled: true` to your values file.
+
+#### Deploying
+If you want to restrict SFT to certain nodes, make sure that in your inventory file you have annotated all of the nodes that are able to run sftd workloads with a node label indicating they are to be used, and their external IP, if they are behind a 1:1 firewall (Wire recommends this.).
```

Comment:
Do we have a recommendation page? Then `Wire recommends this.` could be a link to it.